Nightly compilation of `piet-svg` fails.

It's not possible to compile `piet-svg` with the current nightly compiler. This also means that `piet-svg` can't be documented for the purposes of [docs.rs](https://docs.rs/piet-svg/latest/piet_svg/).

```sh
 Documenting pathfinder_simd v0.5.1
error[E0635]: unknown feature `stdsimd`
  --> C:\Users\Strom\.cargo\registry\src\index.crates.io-6f17d22bba15001f\pathfinder_simd-0.5.1\src\lib.rs:12:49
   |
12 | #![cfg_attr(pf_rustc_nightly, feature(simd_ffi, stdsimd))]
   |                                                 ^^^^^^^

For more information about this error, try `rustc --explain E0635`.
error: could not document `pathfinder_simd`
```

Looks like the issue might have been fixed in [pathfinder#548](https://github.com/servo/pathfinder/pull/548) but there is no new release as of right now, with 0.5.2 still containing the `stdsimd` attribute.

Once a new release of `pathfinder_simd` is out, we should make sure our `piet-svg` dependencies mark that as minimum.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -178,14 +178,8 @@ jobs:
         if: contains(matrix.os, 'ubuntu')
 
       - name: install nightly toolchain
-        # NOTE: Currently pinned to 2024-02-01 with '@master'. Move to just '@nightly' when possible.
-        #       Right now that breaks the build due to pathfinder_simd 0.5.2 not compiling with nightly.
-        #       The issue seems to have been fixed in https://github.com/servo/pathfinder/pull/548
-        #       However there is no new published crate version containing that fix, yet.
-        #       See more at https://github.com/linebender/piet/issues/566
-        uses: dtolnay/rust-toolchain@master
+        uses: dtolnay/rust-toolchain@nightly
         with:
-          toolchain: nightly-2024-02-01
           targets: wasm32-unknown-unknown
 
       - name: restore cache

